var/spack/repos/builtin/packages/pacparser/package.py

```diff
@@ -14,16 +14,19 @@ class Pacparser(MakefilePackage):
 
     version('1.3.7', sha256='eb48ec2fc202d12a4b882133048c7590329849f32c2285bc4dbe418f29aad249')
 
+    # possible race condition in makefile
+    parallel = False
```

Comment:
I did not see any issues building pacparser in parallel, but I didn't yet test it with clang.

```suggestion
    @property
    def parallel(self):
        return self.spec.satisfies('@1.3.7')
```